lib/ansible/modules/net_tools/haproxy.py

```diff
@@ -67,7 +67,7 @@
       - Desired state of the provided backend host.
     required: true
     default: null
-    choices: [ "enabled", "disabled" ]
+    choices: [ "enabled", "disabled", "drain" ]
```

Comment:
could you add some docs the choice "drain" was added in version "2.4" e.g. 

~~~diff
      - Desired state of the provided backend host.
+     - The choice C(drain) is available since version 2.4.
~~~